src/shared/constants/local-storage.ts

```diff
@@ -4,4 +4,4 @@ export const LOCAL_STORAGE_VISITED_REALM = "taho.visitedRealm"
 export const LOCAL_STORAGE_BRAVE = "taho.brave"
 export const LOCAL_STORAGE_COOKIES = "taho.cookies"
 export const LOCAL_STORAGE_DISPLAYED_REALMS = "taho.displayedRealm"
-export const LOCAL_STORAGE_DISPLAYED_QUESTS = "taho.displayedQuests"
+export const LOCAL_STORAGE_DISPLAYED_CHALLENGES = "taho.displayedChallenges"
```

Comment:
hmmm, that change will be a bit problematic, we need to check if that won't cause the "new challenge" label to show up everywhere 🤔